enterprise/frontend/src/embedding-sdk/components/private/PublicComponentStylesWrapper.tsx

```diff
@@ -45,6 +45,8 @@ export const PublicComponentStylesWrapper = forwardRef<
       dir="ltr"
       // eslint-disable-next-line react/prop-types -- className is in div props :shrugs:
       className={`mb-wrapper ${props.className}`}
+      // Mantine's cssVariablesSelector expects data-mantine-color-scheme to be set on the target element
+      data-mantine-color-scheme="light"
```

Comment:
Mantine implicitly expects `data-mantine-color-scheme` to be present on element where `cssVariablesSelector` is applied to. Took me a long time to find out why the styles is broken.